modules/ROOT/pages/create-reference-map.adoc

```diff
@@ -0,0 +1,163 @@
+= Creating Reference Maps for Message Types
+
+When you create a new message type, you can associate it with a DataWeave reference map. This is mandatory for Receive from Partners message types formatted with JSON or XML and for all Source at Host message types.
+
+* The input of the map is the payload that follows the schema uploaded in the definition for the message type.
+* The output of the map contains the following fields:
+
+|===
+| *Field* | *Description* | *Required*
+| partnerReferenceId
+| Identifier that uniquely identifies the partner
+a| Yes. This field is required for:
+
+* Receive from Partners message types formatted with JSON or XML
+* All Source at Host message types
+| hostReferenceId
+| Identifier that uniquely identifies the host
+a| No. This field is optional for:
+
+* Receive from partners message types formatted with JSON or XML
+* All Source at Host message types
+| BusinessDocumentKey
+|Key reference value, such as `Order Number` or `Invoice number`
+| No
+| customAttributes
+| Array of attributes to extract from the payload.
+{sp}+
+{sp}+
+Use an array, even if there is only one custom attribute to extract. This enables support for transactions with one-to-many relationships, such as transactions in which one shipment notice includes multiple purchase order numbers.
+a| No. This field is optional for:
+
+* All Receive from Partners message types
+* All Source at Host message types
+|===
```

Comment:
We need to have a small sub-section explaining the fields within customAttributes. It is explained here: https://docs.google.com/document/d/1ns_4tVufRIYC_18S-obz7f3eVOkkOErb2Q-HJNWI_58/edit#heading=h.gzxdygdyhas1